sfs/array.py

```diff
@@ -422,39 +459,97 @@ def cube(N, spacing, center=[0, 0, 0], orientation=[1, 0, 0]):
     return ArrayData(positions, directions, weights)
 
 
-def sphere_load(fname, radius, center=[0, 0, 0]):
-    """Spherical secondary source distribution loaded from datafile.
+def sphere_load(file, radius, center=[0, 0, 0]):
+    """Load spherical secondary source distribution from file.
 
-    ASCII Format (see MATLAB SFS Toolbox) with 4 numbers (3 position, 1
-    weight) per secondary source located on the unit circle.
+    ASCII Format (see MATLAB SFS Toolbox) with 4 numbers (3 for the cartesian 
+    position vector, 1 for the integration weight) per secondary source located 
+    on the unit circle which is resized by the given radius and shifted to the
+    given center.
 
     Returns
     -------
     `ArrayData`
         Positions, orientations and weights of secondary sources.
 
+    Examples
+    --------
+    content of `example_array_6LS_3D.txt` 
```

Comment:
sfs/array.py:477:42: W291 trailing whitespace

Reply:
fixed.
Thanks for suggesting, I usually use PyCharm, but not on these corrections. Enabled this option now.